src/data/list/basic.lean

```diff
@@ -3382,6 +3382,23 @@ begin
       simp only [h, prefix_cons_inj, hl, map] } },
 end
 
+lemma filter_map_prefix {l l' : list α} (f : α → option β) (h : l <+: l') :
```

Comment:
If you call it `is_prefix.filter_map`, then it will be possible to write `h.filter_map f`.
```suggestion
lemma is_prefix.filter_map {l l' : list α} (h : l <+: l') (f : α → option β) :
```